cli/azd/pkg/infra/provisioning/terraform/terraform_provider.go

```diff
@@ -4,12 +4,6 @@
 // Package terraform contains an implementation of provider.Provider for Terraform. This
 // provider is registered for use when this package is imported, and can be imported for
 // side effects only to register the provider, e.g.:
-//
```

Comment:
Another instance of a case where you can delete more of the comment if you want.